paddle/operators/dynamic_recurrent_op.h

```diff
@@ -0,0 +1,162 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
+#pragma once
+
+#ifdef PADDLE_WITH_TESTING
+#include "gtest/gtest.h"
+#endif
+
+#include "paddle/framework/lod_tensor.h"
+#include "paddle/framework/operator.h"
+#include "paddle/framework/tensor_array.h"
+#include "paddle/framework/variable.h"
+#include "paddle/operators/rnn/recurrent_op_utils.h"
+
+namespace paddle {
+namespace operators {
+
+using framework::Scope;
```

Comment:
There are two problems:
1. You may not use a using-directive to make all names from a namespace available.
2. Do not use Namespace aliases at namespace scope in header files

refs: https://google.github.io/styleguide/cppguide.html#Namespaces

Reply:
`Scope` is not a namespace, so it should be ok.  

Google-style just tells that`using namespace xxx` is forbidden.

Reply:
`using` is not recommended to use in header file because it may pollute other files that include it, put into cpp file is Ok